ansible/roles/openhpc_runtime/tasks/runtime.yml

```diff
@@ -1,4 +1,14 @@
 ---
+- name: Add OpenHPC updates repository
+  yum_repository:
+    name: OpenHPC-updates
+    description: "OpenHPC-1.3-udpates"
+    baseurl: http://build.openhpc.community/OpenHPC:/1.3/updates/CentOS_7/
+    gpgcheck: yes
+    gpgkey: http://build.openhpc.community/OpenHPC:/1.3/updates/CentOS_7/repodata/repomd.xml.key
+    enabled: yes
+    state: present
```

Comment:
Should we really be adding this into the image? Works for now, but it does tie the role to a specific version of OpenHPC.

Reply:
I have moved this over to stackhpc-image-elements